### PR TITLE
runtests.py: re-enable parallel run and fix on Windows

### PR DESCRIPTION
**Pull request by jianchun**

### CODE REVIEW

**Comment by jianchun** [comment]
@dotnet-bot test OSX osx_osx_release_static please

**Comment by jianchun** [comment]
@mrkmarron @obastemur @digitalinfinity Please review, thanks!

File: test/runtests.py [hunk 1]
```diff
@@ -535,19 +544,22 @@ def main():
         args.folders = [f for f in sorted(files) if not os.path.isfile(f)]
 
     # Set the right timezone, the tests need Pacific Standard Time
-    os.environ['TZ'] = 'US/Pacific'
-    time.tzset()
+    if hasattr(time, 'tzset'):  # only available on Unix
```
Comment (obastemur) [inline test/runtests.py]: Can you please add a 'todo' note here? I think we can have a workaround for Windows here.

**Comment by obastemur** [comment]
LGTM

**Comment by jianchun** [comment]
@obastemur Thanks for review! Updated.

**Pull request merged**